R/class_Fragility.r

```diff
@@ -22,15 +22,22 @@ setMethod("$<-", "Fragility", function(x, name, value) {
 
 
 ## Define the print method
+# setMethod("show", "Fragility", function(object) {
+#     cat("Fragility object:\n")
+#     printSlotValue(object, "ieegts")
+#     printSlotValue(object, "adj")
+#     printSlotValue(object, "frag")
+#     printSlotValue(object, "frag_ranked")
+#     printSlotValue(object, "R2")
+#     printSlotValue(object, "lambdas")
+#     cat("Use '$attr' to access the data\n")
+#     
+#     invisible(object)
+# })
+
 setMethod("show", "Fragility", function(object) {
-    cat("Fragility object:\n")
-    printSlotValue(object, "ieegts")
-    printSlotValue(object, "adj")
-    printSlotValue(object, "frag")
-    printSlotValue(object, "frag_ranked")
-    printSlotValue(object, "R2")
-    printSlotValue(object, "lambdas")
-    cat("Use '$attr' to access the data\n")
-    
+    cat("\nFragility object\n")
+    printSlots(object)
+    cat("Use '@' to access the data\n")
```

Comment:
@ is reserved for developers, please ask users to use $ instead

Suggest edit
`cat("Use '$' to access the slot\n")`